UCSD/campus-mobile
# Remove background BT in build process

## Summary
CI/CD update to remove background BT in build process

## Changelog
[General] [Change] - Removed background BT in ci build process

## Test Plan
- Verify iOS builds as expected
- Verify background BT removed for Android builds
